open-company/open-company-web
# Add desktop app (beta) links to the sidebar menu

~Review and merge #956 before merging this branch, as there will be breaking changes to the Electron JS injections that will need to be fixed here after rebase. See commentary in #956.~

Trello: https://trello.com/c/KdOUOcmq/2148-beta-download-for-desktop-mobile-apps

#### NOTE

The links currently point to a 404! We need to deploy and publish a bumped version of the desktop application for these links to become valid. Why? Because we had been originally naming the DMG/EXE files _with the version included_ (e.g. Carrot-X.Y.Z.dmg), and so linking to "the latest" was not possible. This PR renames the DMG/EXE files to simply Carrot.dmg and Carrot.exe respectively.

### To test

- Open the web app on Mac
- [ ] Do you see the link to the latest DMG in the side menu?
- Open the web app on Windows
- [ ] Do you see the link to the latest EXE in the side menu?
